editors/atom.md

```diff
@@ -1 +1,2 @@
-See https://github.com/prettier/prettier-atom
+See
+[https://github.com/prettier/prettier-atom](https://github.com/prettier/prettier-atom)
```

Comment:
@ikatyang This one intentional?

Reply:
It seems @lipis is still using the old prettier? The list style in this PR is still `-`, and so does this link.